coalib/results/result_actions/ShowAppliedPatchesAction.py

```diff
@@ -0,0 +1,17 @@
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+
+
+class ShowAppliedPatchesAction(ResultAction):
+
+    SUCCESS_MESSAGE = 'All actions have been applied'
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
+
+    def apply(self,
+              result,
+              original_file_dict,
+              file_diff_dict):
+        """
+        Show Applied (P)atches
```

Comment:
I think `(S)` is a better option than `(P)` because it's more obvious since the verb is "Show".